### PR TITLE
gnu-complexity 1.10

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by DomT4** [comment]
Broken on Mavericks, apparently. We're not having much luck with 10.9 & 10.10 recently 🙈.

**Comment by ilovezfs** [comment]
It's straightforward to fix in an unpleasing way:
```
inreplace "src/opts.def", "no-return;", "" if MacOS.version <= :mavericks
```
@mistydemeo may know a better way, though.

**Comment by MikeMcQuaid** [comment]
@ilovezfs 👍 if that's submitted upstream 😉

**Comment by ilovezfs** [comment]
@MikeMcQuaid I doubt upstream would consider it a bug, unfortunately, but I can pretend otherwise if you'd like :smirk:

**Comment by MikeMcQuaid** [comment]
If we patch it, we file it 😀

**Comment by ilovezfs** [comment]
@MikeMcQuaid Yeah, I fear we're just doing something wrong in this case with respect to setting C++ includes or standards or whatever because no-return *does* appear to be supported on Mavericks, so just removing it is not really a valid fix, but a crappy workaround :(. I'd almost prefer just setting `depends_on :macos => :yosemite`. The only reason I'd consider the hack here is the fact that I believe no-return "only" affects optimization, so isn't especially harmful to needlessly excise.

File: Formula/gnu-complexity.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class GnuComplexity < Formula
   desc "Measures complexity of C source"
   homepage "https://www.gnu.org/software/complexity"
-  url "http://ftpmirror.gnu.org/complexity/complexity-1.5.tar.xz"
-  mirror "https://ftp.gnu.org/gnu/complexity/complexity-1.5.tar.xz"
-  sha256 "1f5194c0dc6e813ea1dc7f36bfd05f15786a8ad6e9b3ab65b53d2263a0d93102"
+  url "http://ftpmirror.gnu.org/complexity/complexity-1.10.tar.xz"
```
Comment (vszakats) [inline Formula/gnu-complexity.rb]: Would you mind updating this to https?

**Pull request closed without merging**